acceptance/tests/connect/connect_proxy_lifecycle_test.go

```diff
@@ -212,3 +213,141 @@ func TestConnectInject_ProxyLifecycleShutdown(t *testing.T) {
 		})
 	}
 }
+
+func TestConnectInject_ProxyLifecycleShutdownJob(t *testing.T) {
+	cfg := suite.Config()
+	ver, err := version.NewVersion("1.2.0")
```

Comment:
This version check and the corresponding cfg.ConsulDataplaneVersion.LessThan is not needed as all the proxy lifecycle changes have been backported to all of the release branches and and all consul-dataplane versions. It should work consul-dataplane/consul-k8s 1.x and up.